tests/PySys/environments/environment_tedge.py

```diff
@@ -0,0 +1,67 @@
+import subprocess
+import time
+
+
+from pysys.basetest import BaseTest
+
+
+class TedgeEnvironment(BaseTest):
+    """Class with helper and convenicence methods for testing tedge"""
+
+    def setup(self):
+        self.sudo = "/usr/bin/sudo"
+        self.tedge = "/usr/bin/tedge"
+        self.tedge_mapper_c8y = "tedge-mapper-c8y"
+        self.sudo = "/usr/bin/sudo"
+        self.systemctl = "/usr/bin/systemctl"
+
+    def wait_if_restarting_mosquitto_too_fast(self):
+        """Make sure we do not restart mosqiotto too fast
```

Comment:
"Make sure we do not restart mosqiotto too often" would be better.

Reply:
I also used frequently here :)